external/storm-kafka-client/src/test/java/org/apache/storm/kafka/bolt/KafkaBoltTest.java

```diff
@@ -37,38 +39,48 @@
 import org.apache.storm.tuple.Tuple;
 import org.junit.Test;
 import org.mockito.ArgumentMatcher;
+import org.mockito.internal.util.MockUtil;
```

Comment:
I would prefer not to use internal classes. I think https://static.javadoc.io/org.mockito/mockito-core/2.20.0/org/mockito/Mockito.html#mocking_details can do the same thing.

Reply:
Nice, didn't know about that one.